src/python/WMComponent/JobAccountant/AccountantWorker.py

```diff
@@ -244,7 +251,17 @@ def __call__(self, parameters):
 
         # Now things done at the end of the job
         # Do what we can with WMBS files
-        self.handleWMBSFiles(self.wmbsFilesToBuild, self.parentageBinds)
+        with ExitStack() as stack:
+            ctx_timer = CodeTimer("### Handling WMBS unmerged files")
+            stack.enter_context(ctx_timer)
+            self.handleWMBSFiles(self.wmbsFilesToBuild, self.parentageBinds)
+
+            filepath = pathlib.Path(self.compDir, 'WMBS_unmerged.json')
+            if not (filepath.exists() and filepath.stat().st_size > 0):
+                f = stack.enter_context(open(filepath, 'w'))
+                content = {'wmbsFilesToBuild': self.wmbsFilesToBuild,
```

Comment:
Dennis, I would suggest saving 2 different files - and probably name them after the variable name. IMO it will be more clear and avoid any mistakes/confusion